### PR TITLE
Prepare for release v1.12.0-rc1

### PR DESCRIPTION
**Pull request by aanm**
See the included CHANGELOG.md for a full list of changes.

### CODE REVIEW

**Comment by aanm** [comment]
the doc CI failure is fixed by https://github.com/cilium/cilium/pull/19394
the smoke test CI failure will likely be fixed by https://github.com/cilium/cilium/pull/19395

**Review by michi-covalent — APPROVED** [review]
🚢

File: AUTHORS [hunk 1]
```diff
@@ -320,10 +339,13 @@ Sarvesh Rangnekar                       sarveshr@google.com
 Scott Albertson                         ascottalbertson@gmail.com
 Sean Winn                               sean@isovalent.com
 Sebastian Wicki                         sebastian@isovalent.com
+Sebastien Prune                         prune@lecentre.net
```
Comment (joestringer) [inline AUTHORS]: I think that this is supposed to be:
```suggestion
Sébastien Thomas                        prune@lecentre.net
```

File: AUTHORS [hunk 2]
```diff
@@ -365,24 +387,28 @@ Valas Valancius                         valas@google.com
...
 Vlad Artamonov                          742047+vladdy@users.noreply.github.com
 Vlad Gorodetsky                         v@gor.io
 Vladimir Nachev                         vladimir.penkov.nachev@gmail.com
 Vlad Ungureanu                          vladu@palantir.com
+Wang Dong                               xdragon007@gmail.com
 Wang Li                                 wangli09@kuaishou.com
 Wayne Haber                             whaber@gitlab.com
 Wazir Ahmed                             wazir@accuknox.com
 Weilong Cui                             cuiwl@google.com
+weizhou.lan@daocloud.io                 weizhou.lan@daocloud.io
```
Comment (joestringer) [inline AUTHORS]: Interesting, why does this one show up as two emails in a row..?

```suggestion
Weizhou Lan                             weizhou.lan@daocloud.io
```

**Review by joestringer — COMMENTED** [review]
Just a couple of minor nits in the authors, should be fixable in `.mailmap`

**Review by joestringer — APPROVED** [review]

**Pull request merged**